apps/cli/src/actions/deploy/deployExercise.ts

```diff
@@ -0,0 +1,45 @@
+import { CodingExercise } from "@link-to-code/types";
+import axios from "axios";
+
+const formatApiUrl = (apiUrl: string) => (!apiUrl.endsWith("/") ? apiUrl : apiUrl.slice(0, -1));
+
+export type DeployExerciseOptions = {
+  dryRun: boolean;
+  apiUrl: string;
+  codingExercise: CodingExercise;
+  token: string;
+};
+
+export default async function ({
+  dryRun,
+  apiUrl,
+  codingExercise,
+  token,
+}: DeployExerciseOptions): Promise<string | null> {
+  if (dryRun) return null;
+  try {
+    const { data } = await axios.post<{ permalink: string }>(
+      `${formatApiUrl(apiUrl)}/coding-exercise-templates`,
```

Comment:
Should this have a /api before the coding exercise templates path, or where you thinking to have it as part of the input? It seems as it might be counter intuitive having to remember to also add the base api path in the cli prompt, rather than just the hostname...

Reply:
I migrated the api url to the init command, so the publish command now get it from the settings file. I think in this way usage of the CLI commands is easier.
I've added also `/api` but i still not found it in the functions code 🤔
Where it is set as root path?